# Internal patch for Studio C++ 12.4.
# The upstream project at the ASF is defunct.
--- stdcxx-4.2.1/src/collate.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/collate.cpp	Wed Jun  3 09:21:20 2015
@@ -1236,3 +1236,19 @@
 
 
 }   // namespace std
+#ifndef __LP64__
+#pragma weak "__1cDstdHcollate4Cc_Kdo_compare6kMpkcp244_i_" = "__1cDstdHcollate4Cc_Kdo_compare6kMpkc333_i_"
+#pragma weak "__1cDstdHcollate4Cc_Mdo_transform6kMpkcp2_n0AMbasic_string4Ccn0ALchar_traits4Cc__n0AJallocator4Cc_____" = "__1cDstdHcollate4Cc_Mdo_transform6kMpkc3_n0AMbasic_string4Ccn0ALchar_traits4Cc__n0AJallocator4Cc_____"
+#pragma weak "__1cDstdHcollate4Cc_Hdo_hash6kMpkcp2_l_" = "__1cDstdHcollate4Cc_Hdo_hash6kMpkc3_l_"
+#pragma weak "__1cDstdHcollate4Cw_Kdo_compare6kMpkwp244_i_" = "__1cDstdHcollate4Cw_Kdo_compare6kMpkw333_i_"
+#pragma weak "__1cDstdHcollate4Cw_Mdo_transform6kMpkwp2_n0AMbasic_string4Cwn0ALchar_traits4Cw__n0AJallocator4Cw_____" = "__1cDstdHcollate4Cw_Mdo_transform6kMpkw3_n0AMbasic_string4Cwn0ALchar_traits4Cw__n0AJallocator4Cw_____"
+#pragma weak "__1cDstdHcollate4Cw_Hdo_hash6kMpkwp2_l_" = "__1cDstdHcollate4Cw_Hdo_hash6kMpkw3_l_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdHcollate4Cc_Kdo_compare6kMpkcp244_i_" = "__1cDstdHcollate4Cc_Kdo_compare6kMpkc333_i_"
+#pragma weak "__1cDstdHcollate4Cc_Mdo_transform6kMpkcp2_n0AMbasic_string4Ccn0ALchar_traits4Cc__n0AJallocator4Cc_____" = "__1cDstdHcollate4Cc_Mdo_transform6kMpkc3_n0AMbasic_string4Ccn0ALchar_traits4Cc__n0AJallocator4Cc_____"
+#pragma weak "__1cDstdHcollate4Cc_Hdo_hash6kMpkcp2_l_" = "__1cDstdHcollate4Cc_Hdo_hash6kMpkc3_l_"
+#pragma weak "__1cDstdHcollate4Cw_Kdo_compare6kMpkwp244_i_" = "__1cDstdHcollate4Cw_Kdo_compare6kMpkw333_i_"
+#pragma weak "__1cDstdHcollate4Cw_Mdo_transform6kMpkwp2_n0AMbasic_string4Cwn0ALchar_traits4Cw__n0AJallocator4Cw_____" = "__1cDstdHcollate4Cw_Mdo_transform6kMpkw3_n0AMbasic_string4Cwn0ALchar_traits4Cw__n0AJallocator4Cw_____"
+#pragma weak "__1cDstdHcollate4Cw_Hdo_hash6kMpkwp2_l_" = "__1cDstdHcollate4Cw_Hdo_hash6kMpkw3_l_"
+#endif
--- stdcxx-4.2.1/src/ctype.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/ctype.cpp	Wed Jun  3 09:21:20 2015
@@ -968,3 +968,21 @@
 
 
 }   // namespace std
+#ifndef __LP64__
+#pragma weak "__1cDstdFctype4Cc_Cis6kMpkcp2pnE__rwP__rw_ctype_mask__4_" = "__1cDstdFctype4Cc_Cis6kMpkc3pnE__rwP__rw_ctype_mask__3_"
+#pragma weak "__1cDstdFctype4Cc_Hscan_is6kMnE__rwP__rw_ctype_mask_pkcp4_6_" = "__1cDstdFctype4Cc_Hscan_is6kMnE__rwP__rw_ctype_mask_pkc5_5_"
+#pragma weak "__1cDstdFctype4Cc_Iscan_not6kMnE__rwP__rw_ctype_mask_pkcp4_6_" = "__1cDstdFctype4Cc_Iscan_not6kMnE__rwP__rw_ctype_mask_pkc5_5_"
+#pragma weak "__1cDstdFctype4Cc_Kdo_toupper6kMpcpkc_p3_" = "__1cDstdFctype4Cc_Kdo_toupper6kMpcpkc_4_"
+#pragma weak "__1cDstdFctype4Cc_Kdo_tolower6kMpcpkc_p3_" = "__1cDstdFctype4Cc_Kdo_tolower6kMpcpkc_4_"
+#pragma weak "__1cDstdFctype4Cc_Ido_widen6kMpkcp2pc_4_" = "__1cDstdFctype4Cc_Ido_widen6kMpkc3pc_3_"
+#pragma weak "__1cDstdFctype4Cc_Jdo_narrow6kMpkcp2cpc_4_" = "__1cDstdFctype4Cc_Jdo_narrow6kMpkc3cpc_3_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdFctype4Cc_Cis6kMpkcp2pnE__rwP__rw_ctype_mask__4_" = "__1cDstdFctype4Cc_Cis6kMpkc3pnE__rwP__rw_ctype_mask__3_"
+#pragma weak "__1cDstdFctype4Cc_Hscan_is6kMnE__rwP__rw_ctype_mask_pkcp4_6_" = "__1cDstdFctype4Cc_Hscan_is6kMnE__rwP__rw_ctype_mask_pkc5_5_"
+#pragma weak "__1cDstdFctype4Cc_Iscan_not6kMnE__rwP__rw_ctype_mask_pkcp4_6_" = "__1cDstdFctype4Cc_Iscan_not6kMnE__rwP__rw_ctype_mask_pkc5_5_"
+#pragma weak "__1cDstdFctype4Cc_Kdo_toupper6kMpcpkc_p3_" = "__1cDstdFctype4Cc_Kdo_toupper6kMpcpkc_4_"
+#pragma weak "__1cDstdFctype4Cc_Kdo_tolower6kMpcpkc_p3_" = "__1cDstdFctype4Cc_Kdo_tolower6kMpcpkc_4_"
+#pragma weak "__1cDstdFctype4Cc_Ido_widen6kMpkcp2pc_4_" = "__1cDstdFctype4Cc_Ido_widen6kMpkc3pc_3_"
+#pragma weak "__1cDstdFctype4Cc_Jdo_narrow6kMpkcp2cpc_4_" = "__1cDstdFctype4Cc_Jdo_narrow6kMpkc3cpc_3_"
+#endif
--- stdcxx-4.2.1/src/ti_wtime_put.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/ti_wtime_put.cpp	Wed Jun  3 09:21:20 2015
@@ -51,3 +51,9 @@
 _RWSTD_SPECIALIZE_USE_FACET (wtime_put);
 
 #endif   // _RWSTD_NO_WCHAR_T
+#ifndef __LP64__
+#pragma weak "__1cDstdItime_put4Cwn0ATostreambuf_iterator4Cwn0ALchar_traits4Cw_____Dput6kMn0C_rn0AIios_base_wpkn0ACtm_pkwp8_2_" = "__1cDstdItime_put4Cwn0ATostreambuf_iterator4Cwn0ALchar_traits4Cw_____Dput6kMn0C_rn0AIios_base_wpkn0ACtm_pkw9A_2_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdItime_put4Cwn0ATostreambuf_iterator4Cwn0ALchar_traits4Cw_____Dput6kMn0C_rn0AIios_base_wpkn0ACtm_pkwp8_2_" = "__1cDstdItime_put4Cwn0ATostreambuf_iterator4Cwn0ALchar_traits4Cw_____Dput6kMn0C_rn0AIios_base_wpkn0ACtm_pkw9A_2_"
+#endif
--- stdcxx-4.2.1/src/ti_money_put.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/ti_money_put.cpp	Wed Jun  3 09:21:20 2015
@@ -48,3 +48,9 @@
 _RWSTD_SPECIALIZE_FACET_ID (money_put, TARGS_C);
 _RWSTD_DEFINE_FACET_FACTORY (static, money_put, TARGS_C, money_put);
 _RWSTD_SPECIALIZE_USE_FACET (money_put);
+#ifndef __LP64__
+#pragma weak "__1cDstdJmoney_put4Ccn0ATostreambuf_iterator4Ccn0ALchar_traits4Cc_____G_C_put6kMn0C_irn0AIios_base_cpkcIip5I_2_" = "__1cDstdJmoney_put4Ccn0ATostreambuf_iterator4Ccn0ALchar_traits4Cc_____G_C_put6kMn0C_irn0AIios_base_cpkcIi6I_2_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdJmoney_put4Ccn0ATostreambuf_iterator4Ccn0ALchar_traits4Cc_____G_C_put6kMn0C_irn0AIios_base_cpkcLip5L_2_" = "__1cDstdJmoney_put4Ccn0ATostreambuf_iterator4Ccn0ALchar_traits4Cc_____G_C_put6kMn0C_irn0AIios_base_cpkcLi6L_2_"
+#endif
--- stdcxx-4.2.1/src/wctype.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/wctype.cpp	Wed Jun  3 09:21:20 2015
@@ -1109,3 +1109,31 @@
 _RWSTD_SPECIALIZE_USE_FACET (wctype);
 
 #endif   // _RWSTD_NO_WCHAR_T
+#ifndef __LP64__
+#pragma weak "__1cDstdFctype4Cw_Fdo_is6kMpkwp2pnE__rwP__rw_ctype_mask__4_" = "__1cDstdFctype4Cw_Fdo_is6kMpkw3pnE__rwP__rw_ctype_mask__3_"
+#pragma weak "__1cDstdFctype4Cw_Kdo_toupper6kMpwpkw_p3_" = "__1cDstdFctype4Cw_Kdo_toupper6kMpwpkw_4_"
+#pragma weak "__1cDstdFctype4Cw_Kdo_tolower6kMpwpkw_p3_" = "__1cDstdFctype4Cw_Kdo_tolower6kMpwpkw_4_"
+#pragma weak "__1cDstdFctype4Cw_Kdo_scan_is6kMnE__rwP__rw_ctype_mask_pkwp4_6_" = "__1cDstdFctype4Cw_Kdo_scan_is6kMnE__rwP__rw_ctype_mask_pkw5_5_"
+#pragma weak "__1cDstdFctype4Cw_Ldo_scan_not6kMnE__rwP__rw_ctype_mask_pkwp4_6_" = "__1cDstdFctype4Cw_Ldo_scan_not6kMnE__rwP__rw_ctype_mask_pkw5_5_"
+#pragma weak "__1cDstdFctype4Cw_Jdo_narrow6kMpkwp2cpc_4_" = "__1cDstdFctype4Cw_Jdo_narrow6kMpkw3cpc_3_"
+#pragma weak "__1cDstdMctype_byname4Cw_Fdo_is6kMpkwp2pnE__rwP__rw_ctype_mask__4_" = "__1cDstdMctype_byname4Cw_Fdo_is6kMpkw3pnE__rwP__rw_ctype_mask__3_"
+#pragma weak "__1cDstdMctype_byname4Cw_Kdo_toupper6kMpwpkw_p3_" = "__1cDstdMctype_byname4Cw_Kdo_toupper6kMpwpkw_4_"
+#pragma weak "__1cDstdMctype_byname4Cw_Kdo_tolower6kMpwpkw_p3_" = "__1cDstdMctype_byname4Cw_Kdo_tolower6kMpwpkw_4_"
+#pragma weak "__1cDstdMctype_byname4Cw_Kdo_scan_is6kMnE__rwP__rw_ctype_mask_pkwp4_6_" = "__1cDstdMctype_byname4Cw_Kdo_scan_is6kMnE__rwP__rw_ctype_mask_pkw5_5_"
+#pragma weak "__1cDstdMctype_byname4Cw_Ldo_scan_not6kMnE__rwP__rw_ctype_mask_pkwp4_6_" = "__1cDstdMctype_byname4Cw_Ldo_scan_not6kMnE__rwP__rw_ctype_mask_pkw5_5_"
+#pragma weak "__1cDstdMctype_byname4Cw_Jdo_narrow6kMpkwp2cpc_4_" = "__1cDstdMctype_byname4Cw_Jdo_narrow6kMpkw3cpc_3_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdFctype4Cw_Fdo_is6kMpkwp2pnE__rwP__rw_ctype_mask__4_" = "__1cDstdFctype4Cw_Fdo_is6kMpkw3pnE__rwP__rw_ctype_mask__3_"
+#pragma weak "__1cDstdFctype4Cw_Kdo_toupper6kMpwpkw_p3_" = "__1cDstdFctype4Cw_Kdo_toupper6kMpwpkw_4_"
+#pragma weak "__1cDstdFctype4Cw_Kdo_tolower6kMpwpkw_p3_" = "__1cDstdFctype4Cw_Kdo_tolower6kMpwpkw_4_"
+#pragma weak "__1cDstdFctype4Cw_Kdo_scan_is6kMnE__rwP__rw_ctype_mask_pkwp4_6_" = "__1cDstdFctype4Cw_Kdo_scan_is6kMnE__rwP__rw_ctype_mask_pkw5_5_"
+#pragma weak "__1cDstdFctype4Cw_Ldo_scan_not6kMnE__rwP__rw_ctype_mask_pkwp4_6_" = "__1cDstdFctype4Cw_Ldo_scan_not6kMnE__rwP__rw_ctype_mask_pkw5_5_"
+#pragma weak "__1cDstdFctype4Cw_Jdo_narrow6kMpkwp2cpc_4_" = "__1cDstdFctype4Cw_Jdo_narrow6kMpkw3cpc_3_"
+#pragma weak "__1cDstdMctype_byname4Cw_Fdo_is6kMpkwp2pnE__rwP__rw_ctype_mask__4_" = "__1cDstdMctype_byname4Cw_Fdo_is6kMpkw3pnE__rwP__rw_ctype_mask__3_"
+#pragma weak "__1cDstdMctype_byname4Cw_Kdo_toupper6kMpwpkw_p3_" = "__1cDstdMctype_byname4Cw_Kdo_toupper6kMpwpkw_4_"
+#pragma weak "__1cDstdMctype_byname4Cw_Kdo_tolower6kMpwpkw_p3_" = "__1cDstdMctype_byname4Cw_Kdo_tolower6kMpwpkw_4_"
+#pragma weak "__1cDstdMctype_byname4Cw_Kdo_scan_is6kMnE__rwP__rw_ctype_mask_pkwp4_6_" = "__1cDstdMctype_byname4Cw_Kdo_scan_is6kMnE__rwP__rw_ctype_mask_pkw5_5_"
+#pragma weak "__1cDstdMctype_byname4Cw_Ldo_scan_not6kMnE__rwP__rw_ctype_mask_pkwp4_6_" = "__1cDstdMctype_byname4Cw_Ldo_scan_not6kMnE__rwP__rw_ctype_mask_pkw5_5_"
+#pragma weak "__1cDstdMctype_byname4Cw_Jdo_narrow6kMpkwp2cpc_4_" = "__1cDstdMctype_byname4Cw_Jdo_narrow6kMpkw3cpc_3_"
+#endif
--- stdcxx-4.2.1/src/codecvt.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/codecvt.cpp	Wed Jun  3 09:21:20 2015
@@ -280,3 +280,13 @@
 _RWSTD_SPECIALIZE_USE_FACET (codecvt);
 
 
+#ifndef __LP64__
+#pragma weak "__1cDstdHcodecvt4CcCcnL__mbstate_t__Gdo_out6kMrn0B_pkcp4r6pc8r8_n0AMcodecvt_baseGresult__" = "__1cDstdHcodecvt4CcCcnL__mbstate_t__Gdo_out6kMrn0B_pkc5r5pc7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdHcodecvt4CcCcnL__mbstate_t__Fdo_in6kMrn0B_pkcp4r6pc8r8_n0AMcodecvt_baseGresult__" = "__1cDstdHcodecvt4CcCcnL__mbstate_t__Fdo_in6kMrn0B_pkc5r5pc7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdHcodecvt4CcCcnL__mbstate_t__Jdo_length6kMrn0B_pkcp4I_i_" = "__1cDstdHcodecvt4CcCcnL__mbstate_t__Jdo_length6kMrn0B_pkc5I_i_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdHcodecvt4CcCcnL__mbstate_t__Gdo_out6kMrn0B_pkcp4r6pc8r8_n0AMcodecvt_baseGresult__" = "__1cDstdHcodecvt4CcCcnL__mbstate_t__Gdo_out6kMrn0B_pkc5r5pc7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdHcodecvt4CcCcnL__mbstate_t__Fdo_in6kMrn0B_pkcp4r6pc8r8_n0AMcodecvt_baseGresult__" = "__1cDstdHcodecvt4CcCcnL__mbstate_t__Fdo_in6kMrn0B_pkc5r5pc7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdHcodecvt4CcCcnL__mbstate_t__Jdo_length6kMrn0B_pkcp4L_i_" = "__1cDstdHcodecvt4CcCcnL__mbstate_t__Jdo_length6kMrn0B_pkc5L_i_"
+#endif
--- stdcxx-4.2.1/src/ti_time_put.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/ti_time_put.cpp	Wed Jun  3 09:21:20 2015
@@ -47,3 +47,9 @@
 _RWSTD_SPECIALIZE_FACET_ID (time_put, TARGS_C);
 _RWSTD_DEFINE_FACET_FACTORY (static, time_put, TARGS_C, time_put);
 _RWSTD_SPECIALIZE_USE_FACET (time_put);
+#ifndef __LP64__
+#pragma weak "__1cDstdItime_put4Ccn0ATostreambuf_iterator4Ccn0ALchar_traits4Cc_____Dput6kMn0C_rn0AIios_base_cpkn0ACtm_pkcp8_2_" = "__1cDstdItime_put4Ccn0ATostreambuf_iterator4Ccn0ALchar_traits4Cc_____Dput6kMn0C_rn0AIios_base_cpkn0ACtm_pkc9A_2_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdItime_put4Ccn0ATostreambuf_iterator4Ccn0ALchar_traits4Cc_____Dput6kMn0C_rn0AIios_base_cpkn0ACtm_pkcp8_2_" = "__1cDstdItime_put4Ccn0ATostreambuf_iterator4Ccn0ALchar_traits4Cc_____Dput6kMn0C_rn0AIios_base_cpkn0ACtm_pkc9A_2_"
+#endif
--- stdcxx-4.2.1/src/ti_wtime_get.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/ti_wtime_get.cpp	Wed Jun  3 09:21:20 2015
@@ -51,3 +51,9 @@
 _RWSTD_SPECIALIZE_USE_FACET (wtime_get);
 
 #endif   // _RWSTD_NO_WCHAR_T
+#ifndef __LP64__
+#pragma weak "__1cDstdItime_get4Cwn0ATistreambuf_iterator4Cwn0ALchar_traits4Cw_____Dget6kMn0C_2rn0AIios_base_rnE__rwM__rw_iostate_pn0ACtm_pkwp9B_2_" = "__1cDstdItime_get4Cwn0ATistreambuf_iterator4Cwn0ALchar_traits4Cw_____Dget6kMn0C_2rn0AIios_base_rnE__rwM__rw_iostate_pn0ACtm_pkw9C_2_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdItime_get4Cwn0ATistreambuf_iterator4Cwn0ALchar_traits4Cw_____Dget6kMn0C_2rn0AIios_base_rnE__rwM__rw_iostate_pn0ACtm_pkwp9B_2_" = "__1cDstdItime_get4Cwn0ATistreambuf_iterator4Cwn0ALchar_traits4Cw_____Dget6kMn0C_2rn0AIios_base_rnE__rwM__rw_iostate_pn0ACtm_pkw9C_2_"
+#endif
--- stdcxx-4.2.1/src/ti_time_get.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/ti_time_get.cpp	Wed Jun  3 09:21:20 2015
@@ -47,3 +47,9 @@
 _RWSTD_SPECIALIZE_FACET_ID (time_get, TARGS_C);
 _RWSTD_DEFINE_FACET_FACTORY (static, time_get, TARGS_C, time_get);
 _RWSTD_SPECIALIZE_USE_FACET (time_get);
+#ifndef __LP64__
+#pragma weak "__1cDstdItime_get4Ccn0ATistreambuf_iterator4Ccn0ALchar_traits4Cc_____Dget6kMn0C_2rn0AIios_base_rnE__rwM__rw_iostate_pn0ACtm_pkcp9B_2_" = "__1cDstdItime_get4Ccn0ATistreambuf_iterator4Ccn0ALchar_traits4Cc_____Dget6kMn0C_2rn0AIios_base_rnE__rwM__rw_iostate_pn0ACtm_pkc9C_2_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdItime_get4Ccn0ATistreambuf_iterator4Ccn0ALchar_traits4Cc_____Dget6kMn0C_2rn0AIios_base_rnE__rwM__rw_iostate_pn0ACtm_pkcp9B_2_" = "__1cDstdItime_get4Ccn0ATistreambuf_iterator4Ccn0ALchar_traits4Cc_____Dget6kMn0C_2rn0AIios_base_rnE__rwM__rw_iostate_pn0ACtm_pkc9C_2_"
+#endif
--- stdcxx-4.2.1/src/wcodecvt.cpp	Wed Jun  3 09:21:20 2015
+++ stdcxx-4.2.1/src/wcodecvt.cpp	Wed Jun  3 09:21:20 2015
@@ -1765,3 +1765,19 @@
 _RWSTD_SPECIALIZE_USE_FACET (wcodecvt);
 
 #endif   // _RWSTD_NO_WCHAR_T
+#ifndef __LP64__
+#pragma weak "__1cDstdHcodecvt4CwCcnL__mbstate_t__Gdo_out6kMrn0B_pkwp4r6pc8r8_n0AMcodecvt_baseGresult__" = "__1cDstdHcodecvt4CwCcnL__mbstate_t__Gdo_out6kMrn0B_pkw5r5pc7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdHcodecvt4CwCcnL__mbstate_t__Fdo_in6kMrn0B_pkcp4r6pw8r8_n0AMcodecvt_baseGresult__" = "__1cDstdHcodecvt4CwCcnL__mbstate_t__Fdo_in6kMrn0B_pkc5r5pw7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdHcodecvt4CwCcnL__mbstate_t__Jdo_length6kMrn0B_pkcp4I_i_" = "__1cDstdHcodecvt4CwCcnL__mbstate_t__Jdo_length6kMrn0B_pkc5I_i_"
+#pragma weak "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Fdo_in6kMrn0B_pkcp4r6pw8r8_n0AMcodecvt_baseGresult__" = "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Fdo_in6kMrn0B_pkc5r5pw7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Gdo_out6kMrn0B_pkwp4r6pc8r8_n0AMcodecvt_baseGresult__" = "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Gdo_out6kMrn0B_pkw5r5pc7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Jdo_length6kMrn0B_pkcp4I_i_" = "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Jdo_length6kMrn0B_pkc5I_i_"
+#endif
+#if defined(__LP64__) && defined(__sparc)
+#pragma weak "__1cDstdHcodecvt4CwCcnL__mbstate_t__Gdo_out6kMrn0B_pkwp4r6pc8r8_n0AMcodecvt_baseGresult__" = "__1cDstdHcodecvt4CwCcnL__mbstate_t__Gdo_out6kMrn0B_pkw5r5pc7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdHcodecvt4CwCcnL__mbstate_t__Fdo_in6kMrn0B_pkcp4r6pw8r8_n0AMcodecvt_baseGresult__" = "__1cDstdHcodecvt4CwCcnL__mbstate_t__Fdo_in6kMrn0B_pkc5r5pw7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdHcodecvt4CwCcnL__mbstate_t__Jdo_length6kMrn0B_pkcp4L_i_" = "__1cDstdHcodecvt4CwCcnL__mbstate_t__Jdo_length6kMrn0B_pkc5L_i_"
+#pragma weak "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Fdo_in6kMrn0B_pkcp4r6pw8r8_n0AMcodecvt_baseGresult__" = "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Fdo_in6kMrn0B_pkc5r5pw7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Gdo_out6kMrn0B_pkwp4r6pc8r8_n0AMcodecvt_baseGresult__" = "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Gdo_out6kMrn0B_pkw5r5pc7r7_n0AMcodecvt_baseGresult__"
+#pragma weak "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Jdo_length6kMrn0B_pkcp4L_i_" = "__1cDstdOcodecvt_byname4CwCcnL__mbstate_t__Jdo_length6kMrn0B_pkc5L_i_"
+#endif

